app/templates/search/catalogue.html

```diff
@@ -96,13 +96,17 @@
 </div>
 <div class="tna-container">
   <div class="tna-column tna-column--width-1-4 tna-column--width-1-3-medium tna-column--full-small tna-column--full-tiny tna-!--margin-top-l">
-    <h2 class="tna-heading-l">Filters{% if selected_filters %} ({{ selected_filters | length }}){% endif %}</h2>
+    <h2 class="tna-heading-l">Filters</h2>
     {% if selected_filters %}
-    {{ tnaCompoundFilters({
-      'items': selected_filters,
-      'removeAllHref': url('search:catalogue') + '?q=' + request.GET.get('q') + '&display=' + (request.GET.get('display') or ''),
-      'classes': 'tna-!--margin-top-s'
-    }) }}
+        <div class="tna-aside tna-aside--tight tna-background-accent-light tna-!--margin-top-s">
+            <h3 class="tna-form__heading tna-form__heading--m">Active filters {% if selected_filters %} ({{ selected_filters | length }}){% endif %}:</h3>
```

Comment:
I know it might not exactly be clear, but this can just be a normal heading:

```suggestion
            <h3 class="tna-heading-m">Active filters {% if selected_filters %} ({{ selected_filters | length }}){% endif %}:</h3>
```